src/GeositeFramework/js/Plugin.js

```diff
@@ -196,6 +196,14 @@ require(['use!Geosite',
                 } else {
                     this.get('pluginObject').deactivate();
                     this.trigger('plugin:deselected');
+                    // Remove the `.nav-apps-narow` class if the
+                    // de-selecting the plugin has left no plugins
+                    // active.
+                    if ($('.sidebar').filter(function() {
```

Comment:
Much better -- I'll make that change here and in line 422 referred to in the next comment.